### PR TITLE
Chapter intro business ethics

### PR DESCRIPTION
**Pull request by ajluk**
Issue: #2851

### CODE REVIEW

**Review by Stackblocks — APPROVED** [review]

File: styles/designs/cardboard/parts/_titles-components.scss [hunk 1]
```diff
@@ -28,22 +38,40 @@ $H1--Numbered: (
...
+    flex-direction: enum('ValueSet:::OPTIONAL'), //used in Chapters
+    vertical-align: enum('ValueSet:::OPTIONAL'), //used in Chapters
+    z-index: enum('ValueSet:::OPTIONAL'), //used in Chapters
+    margin-right: enum('ValueSet:::OPTIONAL'), //used in Chapters
+    padding-right: enum('ValueSet:::OPTIONAL'), //used in Chapters
+    margin-left: enum('ValueSet:::OPTIONAL'), //used in Chapters
+    padding-left: enum('ValueSet:::OPTIONAL'), //used in Chapters
+    width: enum('ValueSet:::OPTIONAL'), //used in Chapters
+    left: enum('ValueSet:::OPTIONAL'), //used in Chapters
+    margin-top: enum('ValueSet:::OPTIONAL'), //used in Chapters
```
Comment (wilkus27) [inline styles/designs/cardboard/parts/_titles-components.scss:62]: There is so many OPTIONALs here. Based on the our work on previous themes I have a feeling it's very confusing. When we want to reuse some component later it's very hard to find out which property should be used in our case. What do you think to create separate components if we need to change a lot of properties? In this case I would create `H1--ChapterIntro`, `H1--ChapterIntro__Text` etc. and apply as much hard coded values as possible. It's just my point of view. :wink:   I'm really curious what is your opinion. Maybe you like more to have less amount of components which contain REQUIRED and OPTIONAL properties. Let me know what do you think :slightly_smiling_face:

File: styles/designs/cardboard/parts/_chapterintro-components.scss [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    )
+);
+
+$ChapterIntro__Content: empty_wrapper(IntroContent, ' > .intro-text');
+
+$ChapterIntro__Content__Header: (
+    _name: "IntroContentHeader",
+    _subselector: ' > h2[data-type="document-title"]:first-of-type', 
+    _properties: (
+        font-size: enum('ValueSet:::REQUIRED'),
```
Comment (wilkus27) [inline styles/designs/cardboard/parts/_chapterintro-components.scss:61]: Can we hard code it?

**Review by wilkus27 — CHANGES_REQUESTED** [review]

File: styles/designs/cardboard/parts/_titles-components.scss (hunk 1 above)
Comment (ajluk) [reply styles/designs/cardboard/parts/_titles-components.scss:62]: Yes, it is a good idea to create new component for such cases. :slightly_smiling_face:

**Review by wilkus27 — CHANGES_REQUESTED** [review]

**Review by wilkus27 — APPROVED** [review]
Everything looks good! :relaxed: A lot of work is done here. Good job! :+1:

**Pull request merged**